public/assets/images/onboarding/steps/explore-apps-popup.svg

```diff
@@ -0,0 +1,9 @@
da6zw2Cb2InQBswG17Dl3CqFSOXPhCI7SxgUjlLsArmGQcubdoJys79VzAcpd5KPchcPiXgUwwHaTNsz+UdRvuDd7zm0x0cyACbgyxH43BAv9hKpDyEaGBtj1SmpqajF0CLFvhzFcvTrlCoMVfmC4/HTIi/v9dPbUF0mp1lS637HZpwuYDYDhIzZm3f4jAMueg7ktgDYiIgeQzf5BObdgiHIX9dPGBX4BEUASEM69odyTpaSP+AzsyWE9JUxj1ZLgko0MzC1dv14holEGTLLfNbgSUK3Ozp7wkyFxnNIHhuvtdxcwm+Dij8meN9KbPecmZc+ZoJz5Adq4YEgcZ987qdRd4tyguJaThdpsXEQ2AJs7rsA5Dwroo3dfH9GeVCJPGOvTQSu1VrjVydKk8RTtdy0dlMBi1dIA6/VSe3unHjAtJTRTLX29ZaRaRnDpai0T+z3upHGE/R7djWHsELqAMRk2jRtn3zmZk3UrlD13lHLmD4mAGgngsr6j7HkjlDPfp1yb5ydxX9Ztysm6RTniNRTwhjj/3MPjhoBpLmGsXZ1sWrMVe9IY67ZMai0JLg2qCPu9TQassUVMHGuKZbnW6iVAZQSW9e73xGotq3DBxHABswks+WOy/2uyNnvO2A/ZApgAZc8do+w54wIajBuyJpTjeVA3FShxHfcGaUPWdyLeeD6oA0wDKwX2e6KTxlZVS+7G0APWbFJr6VXLyMgwgitatey132UjI5Yb6AImk2Hjcfa9k+9tmP397Q1Zt1VgANY4bZh9hzbM/oE2ZAVpw9yAgE0ZcR1gjdGGrFuElPLpMzc0wDS4kt0yLck1W1YnjTXATFudou33tjYlRbx2vYIaGlXALNVaxkaGMDWQAsrhkP1uddLYBcxGqIw+asPs7ys2zL5F67NGaD2gmn1bxPqsUdowe4LWZ43T+iy/uK68xn24dkfMg+nrrfQyMjSopHTQqNaSVUu23wEY4NIAswSXkg7aplpJ2u+xVEu+5gJmRIXN51ZneX6xLmv4wn2zfRP3zfaR1bjy9ZD6KKDEjAw712zJ3e9W7fdwN0a0anGDLuz31rYOBbBrFdTQ0Ky2OyVmv09FtaZiv8sQxTp2AbMZplgfl3PPzZ+vzxrIEbDd67u8bpaPYoUALJaRIfZ9t+YQOjJpLClXtJERXrOldWVE2O8CsGsVdO1aBdU3NOvmtazWWqkzMvT2eyyo5GsuYLGImOZrPp/vcuT+78l0ZGhrtlJkv5ulhJGTxq2tHQIudJzXNzQpgOm6MaZYa1lcs+XkSmMXsGmGKNaPlwHTwEqJ/R5es8VpoVG9lYxqCRterbUAWPm16yLq65tEL2F4bsu81kp31XIVLNa3Oo2uMWAaXDakhKmw32UjI1K15D5CPPBBAFZ+nQCYFbh0DqHJpHHUo1e17vcO4iZdJ1XLBSyNIIr1RwFgergMut/T0H5nIyPepLEArPw6lcuAGU0aq+1OOrhMG3QTX7MVntNSlvTLgEz12E0RY33Dp/naIACzQbWmsmYrskFXSwttWLOlB6zRtCNDB5ZQremZNE4GNhewaYYo1o8fHARgBqqlW7OV3JZpVtdsGc9rmdvvRqqlW7OlrttCOtiCFFFTsEZ9g66JakVtPhNjyzROB41WGjupWjKILmCxvuHTfE0GTJvbSiYlNNsfI6k+wvj2u2xkdMVYEIkHPjBgdfUSYA52v0eC5fRGny5g0wxRrB/PgJnBpdnv0sYzsVYaJz1prDXpGu/qFMvIkFcbh00MxX7HAx/wBURogMltTmo6GKVaaWC/yyoV69gFLNY3fJqvDQwOXhJwzSDV4pXGaHVqlgGra0hg7/epdr8nN2kcCySzaz8RwCamGZXkfnx//+Ax0+2p02zSWLbfY6kWwOI+Qg2wq9eotq4hoklX3TmXd9A12kV3Gu13M6Aiz/8UAGtqaruc3Dd8mt/l8fh+Fe9RQPEdwqmt2dLanCI2+tTMDEsNusZrtpqbW+nq1WsiNMBM7ffYqqU3MtopkTVbkVDY+fqnAFh9Y1vpNKOS/I8fHPR5p9RHaLTxjNRDaOwSTq3WstpH2BQJmBW4NNXCw+yUieN4k8ZOGxmxgPxJAFbf/ovkv+HT/E5Pf/8xYWZYNDKmzX6P2DJNMTTC6aC8AQ23OukAq61XHqbAKWGa2++xoJKvzXTAMjY9ZK49Hs/P+/sHJ5AKcsRexq/vIzTqIUxMtZLfMo1rLQ2uCIeQAbtytZxqZMAEXLFTwnRWrZ8SYPX1zffwdzVjR4+n/1h62e/2rNlqamolwKUBZqBaqVizJQNh9/FMVrCmprZLGQtV5B/cOzDYEPOhCtM8aSzb75pixdkyrVEGrKaetAcqtCnqlfgDw1Nnv1sFcQYDNlFfX//zyO9pxr5Gquj1DkxE75ERf1enxFLC5LdM04Fl1OokJo7D9rsG2JVyqpEB04wMtfu9JXMfGD5TAZsRqWHkbwOPx/MrQDY1I8O+VidhZEjzWhpgFludGhtb6cqVchEMWOKqZbwfoVWFcfq+mQhYfVN7TuR3c8a8BmQe70ADL4TEOHUjw5pqWbXfrXa/NzS26ACLB1eq+wjtgG+GATYxI5Ur8rcD0sXeXs95M7i0JSVaD2Fi3e+xGnRNVcsoJTSd11LWbDFg314pp+qaem1eSz9pjPQwcx+9OlMAgx1fn8nzXZEQWXkt1MwzcAE1Fkc0XGaTxs7Z70LBTOEK2+8ADHApgNUJwDLFfreqbpkOmALWDLDirQBldo/H4/mFx+Mp7enxXu7t804oT4Y0Vy1d97va6mS2ZivcAR970lhLCy1u9An7vb4BgF0VUV1dR/HgsvqlTqf7MhEwAVVz2/n6+tZfmX3nfvLnu7s99xhH9z3d3fpo7+6+R4v27nvapxDN7e336KK5/Z5mk6ivb763vLxiOaK6un6ufF99c/s94Wi+B7l/JkZ5ecWib8vLlyPS/8+fvO3+/wGwXvpWt3Q7bwAAAABJRU5ErkJggg=="/>
```

Comment:
This is a png inside an svg. Should it be exported differently?

Reply:
I can check on this. I just exported the group as an svg from Figma so not sure why it exported this way ...will check. It seems like I've come across quite a few reported issues with exporting from Figma.

Reply:
@eugeniadigon it looks like the Figma designs use pngs so I can't export svgs correctly. It just wraps the png in an svg. Is this how you've always added illustrations to Figma? Can those be replaced with svgs or should I just use pngs? Am I missing something?

Reply:
First of all - so sorry for any confusion caused by this. The images are originally PNG because of shadowing/texturing.  I'm aware of the advantages of using SVGs images, but I'm not aware of any downside of using PNGs either (especially if they don't need to scale). Is there anything that technically makes it not desirable? If so, I can make modifications but some illustrations will end up being "flat". 

If the problem is how Figma exports PNGs into SVGs, I can export them myself on another software that works better for this and make a compilation of all images. If there are no technical issues with exporting directly PNGs, then let's use PNGs?
Let me know how I can help

Reply:
We can use PNGs if that's the best format for it to be exported as. Just that in this case, we should export as a .png.

Reply:
Thx for the feedback @eugeniadigon, I will switch to using pngs. There is no reason I need to use svgs in this case.

Reply:
Thx for catching this @kyranjamie. I couldn't figure out why the images seemed slightly blurry to me ...now they don't exported as pngs. I thought the code in the export looked off but I didn't understand it was an svg wrapping a png. I do see it now, thx.

Reply:
I have replaced all the onboarding images with pngs so going to resolve this thread.